cms/djangoapps/contentstore/rest_api/v2/views/home.py

```diff
@@ -0,0 +1,91 @@
+"""HomePageCoursesViewV2 APIView for getting content available to the logged in user."""
+import edx_api_doc_tools as apidocs
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from openedx.core.lib.api.view_utils import view_auth_classes
+
+from cms.djangoapps.contentstore.utils import get_course_context_v2
+from cms.djangoapps.contentstore.rest_api.v2.serializers import CourseHomeTabSerializerV2
+
+
+@view_auth_classes(is_authenticated=True)
+class HomePageCoursesViewV2(APIView):
+    """View for getting all courses available to the logged in user."""
+
+    @apidocs.schema(
+        parameters=[
+            apidocs.string_parameter(
+                "org",
```

Comment:
Should there be two ways to filter by `org`? With this parameter and the `search` parameter?

Reply:
Yes. When using the staff-optimized view, turned on by https://github.com/openedx/edx-platform/blob/43747ee8f08046a8892b5ad6cc9534fc0c2a896d/cms/djangoapps/contentstore/views/course.py#L138-L142, the queries done are using this parameter:
![image](https://github.com/openedx/edx-platform/assets/64440265/65e16bcb-de48-40cd-bd2e-461444968515)

So we must maintain it for backwards compatibility.